network-store-client/src/main/java/com/powsybl/network/store/client/ThreeWindingsTransformerImpl.java

```diff
@@ -0,0 +1,405 @@
+/**
+ * Copyright (c) 2020, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.network.store.client;
+
+import com.powsybl.iidm.network.*;
+import com.powsybl.network.store.model.*;
+
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+
+/**
+ * @author Nicolas Noir <nicolas.noir at rte-france.com>
+ */
+public class ThreeWindingsTransformerImpl extends AbstractIdentifiableImpl<ThreeWindingsTransformer, ThreeWindingsTransformerAttributes> implements ThreeWindingsTransformer, CurrentLimitsOwner<ThreeWindingsTransformer.Side> {
```

Comment:
As for injections (see AbstractInjection) and branches (see AbstractBranch) you have to override addExtension(...) to support Connectable position extensions.

Reply:
Done